packages/components/Molecules/Statisticsgrid/statistics-grid.scss

```diff
@@ -0,0 +1,66 @@
+@import "~@dpc-sdp/ripple-global/scss/settings";
+@import "~@dpc-sdp/ripple-global/scss/tools";
+
+$rpl-statistics-grid-background-primary: rpl_color('light_neutral') !default;
+$rpl-statistics-grid-background-secondary: white !default;
+$rpl-statistics-grid-padding: ($rpl-space * 10) ($rpl-space * 4);
+$rpl-statistics-grid-margin: ($rpl-space * 2);
+
+.rpl-statistics-grid {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: flex-start;
+  gap: $rpl-statistics-grid-margin;
+  font-size: 1em;
+
+  &__block {
+    display: flex;
+    flex: 1 0 27%;
```

Comment:
Where does the flex-basis of 27% come from here? Might need a comment for context. Just curious because 4 or 5 columns wouldn't be 27 🤔

Reply:
✨  honestly I'm not sure why 27% works, it's a combination of the grid, breakpoints and constrained body in the layout where it appears - I think the flex calc might be including the sidebar in the total width?